extensions/notebook/src/jupyter/jupyterServerInstallation.ts

```diff
@@ -786,7 +788,7 @@ export class JupyterServerInstallation implements IJupyterServerInstallation {
 		return this._usingConda;
 	}
 
-	private isCondaInstalled(): boolean {
+	public isCondaInstalled(): boolean {
```

Comment:
Instead of making this public you should also just be able to stub out existsSync (which we shouldn't be using...). Preferable not to make stuff public unless we need to.